library/alloc/src/vec/mod.rs

```diff
@@ -216,7 +216,7 @@ mod spec_extend;
 /// # Slicing
 ///
 /// A `Vec` can be mutable. Slices, on the other hand, are read-only objects.
-/// To get a [slice], use [`&`]. Example:
+/// To get a [slice][prim@slice], use [`&`]. Example:
```

Comment:
Just using '&' works for a link !? Intra doc links are even more amazing than I thought

Reply:
Yes, although some people wish it didn't :laughing: https://github.com/rust-lang/rust/pull/80181#discussion_r546935967